blackberggroup/VA-CCPI
# Feat/rl 7420756148/ccpi site edits

## What type of PR is this? (check all applicable)


Client updates to sandbox CCPI content

## Related Tickets & Documents
- Monday Task: https://blackberg-group.monday.com/boards/1931719392/pulses/7420756148

## QA Instructions, Screenshots, Recordings

Verify these changes:
- Swap “pilots” & “strategy” boxes on home page
- Set all photos left aligned on team page
- Updated/Fixed Bailee Bannan’s photo
- Edited “Joseph Wallace“ to “Joseph (Mike) Wallace”
- Remove space (40px) between “establishment” and “alignment” on about page


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- _includes/about/alignment.html
 - _includes/about/establishment.html
 - _includes/home/pilot.html
 - _includes/our-team/team.html
 - _site/about/index.html
 - _site/assets/css/styles.css
 - _site/assets/sass/vha/pages/our-team/_index.scss
 - _site/index.html
 - _site/our-team/index.html
 - assets/css/styles.css
 - assets/sass/vha/pages/our-team/_index.scss
